### DISCUSSION TITLE
Next Releasedate

### DISCUSSION THREAD

**Discussion by mirisbowring** [ossf/scorecard#4016] [Q&A] [answered]
Hi,

currently I am running into a problem that has been fixed in october https://github.com/ossf/scorecard/pull/3596
Unfortunately, this Merge happened after the last release.
Therefor, my question is, when to expect a new release?

Thanks!

**Comment by spencerschrock** [comment, marked as answer]
There is a new release candidate today!
https://github.com/ossf/scorecard/releases/tag/v5.0.0-rc1
